frappe_telegram/client.py

```diff
@@ -2,21 +2,35 @@
 
 import frappe
 from frappe_telegram.handlers.logging import log_outgoing_message
+from telegram.parsemode import ParseMode
 
 """
 The functions defined here is provided to invoke the bot
 without Updaters & Dispatchers. This is helpful for triggering
 bot interactions via Hooks / Controller methods
 """
 
+def send_message(message_text: str, parse_mode=None, user=None, telegram_user=None, from_bot=None):
+    '''
+    Send a message using a bot to a Telegram User
+
+    message_text: `str`
+        A text string between 0 and 4096 characters that will be the message
+    parse_mode: `ParseMode`
+        Choose styling for your message using a ParseMode class constant. Default is `None`
+    '''
+
+    if parse_mode:
+        if parse_mode not in \
+          [value for name, value in vars(ParseMode).items() if not name.startswith('_')]:
```

Comment:
@Abadulrehman this is some next level enum validation 👍🏼 😄 
I think i would have let telegram-bot-server handle the validation 😄 
But since this is here, enum-validation might be useful in other scenarios too.. Can we have a generic Enum Validator function defined in frappe_telegram.utils ?